cookbook/catalog/common/remove.rst

```diff
@@ -0,0 +1,38 @@
+How to Remove Non-Product Objects
+=================================
+
+To remove one or many objects, we provide a dedicated service which provides methods 'remove' and 'removeAll' through the implementation of ``Akeneo\Component\StorageUtils\Remover\RemoverInterface`` and ``Akeneo\Component\StorageUtils\Remover\BulkRemoverInterface``.
+
+Use the Remover to Remove a Single Object
+-----------------------------------------
+
+You can remove one or many objects of a kind with a dedicated service, the remover checks that the used object is supported (for instance, you can't use the attribute remover to remove a family).
+
+We define these different services to ease the future changes and allow you to override only one of them to add a specific business logic.
```

Comment:
As there are one identified service per managed item, you can override only one service (for product, or category..) etc